projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-viewer.component.scss

```diff
@@ -54,3 +54,10 @@
 .annotationLayer * {
   z-index: 2;
 }
+
+mv-pdf-viewer {
+  .commentSummaryHeader {
+    top: 37px
+  }
```

Comment:
is there a reason this is done here and not in the header component?

Reply:
yes this will move out of there soon in separate css file in next pr